lib/email/receiver.rb

```diff
@@ -107,6 +108,7 @@ def create_incoming_email
     def process_internal
       raise BouncedEmailError  if is_bounce?
       raise NoSenderDetectedError if @from_email.blank?
+      raise FromReplyByAddressError if is_from_reply_by_email_address?
```

Comment:
What happens when that exception is raised?

Reply:
I think it gets caught by https://github.com/discourse/discourse/blob/master/lib/email/processor.rb#L59 then handled by https://github.com/discourse/discourse/blob/master/lib/email/processor.rb#L71

Reply:
Oh, I looked at that code more and it looks like it is probably still going to send a rejection mail in that case. I'll work on making it better.